source/overview/corba.rst

```diff
@@ -0,0 +1,22 @@
+10 things you should know about CORBA
+=====================================
+
+1.  **You don’t need to know CORBA to work with TANGO**
+
+2.  CORBA is the acronym for **C**\ ommon **O**\ bject **R**\ equest **B**\ roker **A**\ rchitecture and it is a standard defined by the `Object Management Group (OMG) <www.omg.org>`_
```

Comment:
hehe I tried to compile but I was unsuccessful... this was the error:

# Sphinx version: 1.6.2
# Python version: 2.7.11 (CPython)
# Docutils version: 0.13.1 release
# Jinja2 version: 2.9.6
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "c:\python27\lib\site-packages\sphinx\cmdline.py", line 305, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "c:\python27\lib\site-packages\sphinx\application.py", line 168, in __init__
    confoverrides or {}, self.tags)
  File "c:\python27\lib\site-packages\sphinx\config.py", line 150, in __init__
    execfile_(filename, config)
  File "c:\python27\lib\site-packages\sphinx\util\pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "c:\python27\lib\site-packages\six.py", line 699, in exec_
    exec("""exec _code_ in _globs_, _locs_""")
  File "<string>", line 1, in <module>
  File "conf.py", line 18, in <module>
ImportError: No module named IPython.sphinxext

Of course the problem is that I didn't spend enough time on it.. Sorry about that.